osdep/timer.h

```diff
@@ -45,7 +45,7 @@ int mp_start_hires_timers(int wait_ms);
 void mp_end_hires_timers(int resolution_ms);
 #endif  /* _WIN32 */
 
-#define MP_START_TIME 10 * INT64_C(1000000000)
+#define MP_START_TIME (10 * INT64_C(1000000000))
```

Comment:
It is 10ns, why not use macros? I know it would need to be moved...

Anyway, do we know on which platforms this might even be needed? We use clock_monotonic, qpc and mach, which are strictly monotonic, so unless you start mpv before the kernel I doubt you will ever get 0 or close to it. Maybe we just should remove this implementation detail of our timer?

Reply:
> why not use macros?

You mean the `MP_TIME` macros? Sure, I can change it.

> do we know on which platforms this might even be needed?

Not sure, I haven't dealt with the timer code much.

> Maybe we just should remove this implementation detail of our timer?

Glancing at it, we probably can.

Reply:
> Maybe we just should remove this implementation detail of our timer?

Done

Reply:
Good point. This does seem like a pointless thing to have now.